Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -2598,6 +2602,12 @@ RefPtr<WebPageProxy> webkitWebViewCreateNewPage(WebKitWebView* webView, const Wi
     return &getPage(newWebView);
 }
 
+Ref<API::PageConfiguration> webkitWebViewGetConfigurationForNextRelatedView(WebKitWebView* webView)
+{
+    ASSERT(webView->priv->configurationForNextRelatedView);
+    return *webView->priv->configurationForNextRelatedView;
+}
+
```

Comment:
We don't need this after https://github.com/WebKit/WebKit/pull/28383 lands. With that patch we can do something like this in `webkitWebViewConstructed`:

```
Ref<API::PageConfiguration> pageConfiguration = priv->relatedView && priv->relatedView->priv->configurationForNextRelatedView ? *priv->relatedView->priv->configurationForNextRelatedView : webkitWebViewCreatePageConfiguration(webView);
webkitWebViewCreatePage(webView, WTFMove(pageConfiguration));
```